doc/theoretical_description_conformity_scores.rst

```diff
@@ -109,8 +109,8 @@ Key takeaways
 References
 ==========
 
-[1] Angelopoulos, A. N., & Bates, S. (2021). A gentle introduction to conformal
-prediction and distribution-free uncertainty quantification. arXiv preprint arXiv:2107.07511.
+[1] Lei, J., G'Sell, M., Rinaldo, A., Tibshirani, R. J., & Wasserman, L. (2017). Distribution-Free
```

Comment:
It is best to refer to the published version instead of the pre-print : https://www.tandfonline.com/doi/full/10.1080/01621459.2017.1307116

Reply:
Sure, I'll replace it with a reference to the version you shared in the next commit👍